.codegen/service.py.tmpl

```diff
@@ -312,7 +312,7 @@ self._api.do('{{.Verb}}',
   {{- else if .Response}} -> {{if .Response.ArrayValue -}}
     Iterator[{{template "type" .Response.ArrayValue}}]
   {{- else if .Pagination -}}
-    Iterator[{{.Pagination.Entity.PascalName}}]
+    Iterator[{{template "type" .Pagination.Entity}}]
```

Comment:
do we iterate over `str` and `int` somewhere?

Reply:
Yes: list metastore assignments response is a list of workspace IDs, which are plain ints.